RecoPixelVertexing/PixelTrackFitting/test/PixelTrackTest.cc

```diff
@@ -58,10 +58,10 @@ void PixelTrackTest::analyze(const edm::Event& ev, const edm::EventSetup& es) {
   using namespace std;
   using namespace reco;
 
-  cout << "*** PixelTrackTest, analyze event: " << ev.id() << endl;
+  edm::LogPrint("PixelTrackTest") << "*** PixelTrackTest, analyze event: " << ev.id();
   Handle<SimTrackContainer> simTrks;
   ev.getByLabel("g4SimHits", simTrks);
-  //  std::cout << "simtrks " << simTrks->size() << std::endl;
+  //  std::edm::LogPrint("PixelTrackTest") << "simtrks " << simTrks->size() << std::endl;
```

Comment:
```suggestion
  //  edm::LogPrint("PixelTrackTest") << "simtrks " << simTrks->size();
```
(But ok, it is in a commented out line)